### PR TITLE
Add ROR to footer

### PR DESCRIPTION
**Pull request by asmacdo**
Fixes https://github.com/con/centerforopenneuroscience.org/issues/39

I initially tried to add this as stay-in-touch contact button like github/twitter/email, but it didn't match the ROR display guidelines https://ror.readme.io/docs/display

Since we have room, it is recommended to use the [full ror id](https://ror.readme.io/docs/display#full-ror-id)

### New footer 

image

### Old footer 

image

### CODE REVIEW

**Comment by yarikoptic** [comment]
Thank you! Looks good, I will merge and deploy from local

File: theme/templates/base.html [hunk 1]
```diff
@@ -81,7 +81,11 @@
     <div class="row">
       <div class="col-md-8">
         <h3>Center for Open Neuroscience</h3>
+      <a href="https://ror.org/04tfhh831"><img alt="ROR logo" src="https://raw.githubusercontent.com/ror-community/ror-logos/main/ror-icon-rgb.svg" height="24" /> https://ror.org/04tfhh831</a>
```
Comment (yarikoptic) [inline theme/templates/base.html:84]: although -- could you please add a copy of that icon -- I do not want to rely on external resources... if anything - should have been from some commit or tag so if they move etc, doesn't disappear.  But a copy is better

**Review by yarikoptic — CHANGES_REQUESTED** [review]

**Comment by asmacdo** [comment]
I thought i was losing my mind (couldnt access /theme/static/img/contact/ror-icon-rgb.svg or variations) -- until I realized that the paths in `output` are different from the raw directory structure. Specifically , `output` does not have a static dir.

**Comment by yarikoptic** [comment]
did you check that it works ok with such a relative path?

**Comment by asmacdo** [comment]
Yes

**Pull request merged**